actions/play_youtube_playlist_MOD.js

```diff
@@ -37,66 +39,53 @@ module.exports = {
     const { Audio } = this.getDBM()
     const Mods = this.getMods()
     const url = this.evalMessage(data.url, cache)
-    const ytpl = Mods.require('ytpl')
-    // const moment = Mods.require('moment')
+    let maxvideos = this.evalMessage(data.maxvid, cache)
+    const ytpl = Mods.require('ytpl') // be sure you have the latest YTPL, this was modified with 2.0.3 in mind
     const { msg } = cache
-    const options = {}
-
-    // const re = new RegExp('(^[0-9]?[0-9]:[0-9][0-9]:[0-9][0-9]$)')
-    // const re1 = new RegExp('(^[0-9]?[0-9]:[0-9][0-9]$)')
 
     // Check Input
     if (!url) {
       return console.log('Please insert a playlist url!')
     }
+    if (!maxvideos) {
+      maxvideos = 250
+    }
 
     // Check Options
     if (data.seek) {
-      options.seek = parseInt(this.evalMessage(data.seek, cache))
+      var seek = parseInt(this.evalMessage(data.seek, cache))
     }
     if (data.volume) {
-      options.volume = parseInt(this.evalMessage(data.volume, cache)) / 100
+      var vol = parseInt(this.evalMessage(data.volume, cache)) / 100
```

Comment:
Do not use var. Keep the `options` object and then simply pass the `options` object as a argument in the `Audio.addToQueue` function